plugins/PrivacyManager/PrivacyManager.php

```diff
@@ -206,15 +211,18 @@ public function shouldDisablePostProcessing(&$shouldDisable, $request)
     public function onConfigureVisualisation(Plugin\Visualization $view)
     {
         if ($view->requestConfig->getApiModuleToRequest() === 'Referrers' && !$view->requestConfig->idSubtable) {
-            $config = new Config();
+            $idSite = $view->requestConfig->getRequestParam('idsite');
```

Comment:
I think this will default to `string`, and not be an int, so best to add an `else { (int) $idSite }` part to have one complaint less later.